src/modules/Transcript/index.tsx

```diff
@@ -346,7 +375,7 @@ export const TranscriptUI = (props: TranscriptUIProps): ReactElement => {
                     return (
                         (event?.audioRecordingUrl || event?.isLive) && (
                             <Playbar
-                                hideEventDetails={useConfigOptions}
+                                hideEventDetails
```

Comment:
hiding event details in the playbar by default, unless it has the "swap" icon present